src/component/handlers/edit/editOnPaste.js

```diff
@@ -188,6 +211,14 @@ function editOnPaste(editor: DraftEditor, e: SyntheticClipboardEvent): void {
   }
 }
 
+function getHTML(data: DataTransfer) {
+  // Work around DataTransfer issue in IE11 https://github.com/facebook/draft-js/issues/656
+  if (data.data.getData && !data.types.length) {
+    return undefined;
+  }
+  return data.getHTML();
```

Comment:
To your point in slack, do we have a good grasp on when this function will return text? It seems important as we're driving whether to use the paste trap on the html being present or not